Cargo.toml

```
@@ -88,7 +88,7 @@ ra-ap-rustc_lexer = { version = "0.35.0", default-features = false }
 ra-ap-rustc_parse_format = { version = "0.35.0", default-features = false }
 ra-ap-rustc_index = { version = "0.35.0", default-features = false }
 ra-ap-rustc_abi = { version = "0.35.0", default-features = false }
-ra-ap-rustc_pattern_analysis = { version = "0.37.0", default-features = false }
+ra-ap-rustc_pattern_analysis = { version = "0.40.0", default-features = false }
```

```suggestion
ra-ap-rustc_pattern_analysis = { version = "0.41.0", default-features = false }
```

I don't think 0.40 actually includes your fix, and 0.41 will hopefully come out in a couple of hours.

---

I checked, it does include the fix :) It was published shortly after the fix landed, to my own surprise

---

Ah, sorry then. Still, can you bump it to the new version? It's out now.

---

Just saw that ^^ Yeah I'll bump next time it's useful